coordinator/src/main/scala/filodb/coordinator/v2/FiloDbClusterDiscovery.scala

```diff
@@ -71,12 +71,23 @@ class FiloDbClusterDiscovery(settings: FilodbSettings,
   def shardsForLocalhost(numShards: Int): Seq[Int] = shardsForOrdinal(ordinalOfLocalhost, numShards)
 
   lazy private val hostNames = {
-    require(settings.minNumNodes.isDefined, "Minimum Number of Nodes config not provided")
-    if (settings.k8sHostFormat.isDefined) {
-      (0 until settings.minNumNodes.get).map(i => String.format(settings.k8sHostFormat.get, i.toString))
+    require(settings.minNumNodes.isDefined, "[ClusterV2] Minimum Number of Nodes config not provided")
+    if (settings.k8sHostFormat.isDefined && settings.akkaPort.isDefined) {
```

Comment:
The k8sHostFormat config value itself can contain `${akka.remote.netty.tcp.port}` in it - it will be resolved. So code for adding`akkaPort` is not needed. But the current approach is also okay if you want to keep it.

Reply:
Ack, Agree with you. Updated the config to simplify this.